pkg/integration/tests/stash/stash.go

```diff
@@ -29,7 +29,7 @@ var Stash = NewIntegrationTest(NewIntegrationTestArgs{
 
 		t.Views().Stash().
 			Lines(
-				Contains("my stashed file"),
+				MatchesRegexp("\\ds .* my stashed file"),
```

Comment:
Nitpick: it's useful to use backticks instead of double quotes for regular expressions, you don't have to escape the backslash then and it makes the regex easier to read (not a big deal in this simple case, but useful to remember in general).

```suggestion
				MatchesRegexp(`\ds .* my stashed file`),
```